projects/recipe-page/phase-2-css-prompt.md

```diff
@@ -65,14 +66,12 @@ You'll be building on the HTML code you completed in Phase 1 of this project.
 
 #### Part 1 - Create a CSS file in the proper directory of your file tree
 
-1. Using the Terminal, ensure you are in the `recipe-page` directory. Create a file called `styles.css`. You can verify this worked by running the `ls` command.
-2. Use the Terminal shortcut `code .` to open the `recipe-page` directory in VSCode.
-3. Run `git status`. You should see that 'styles.css' is not being tracked yet.
-4. Add and commit, making sure to write a descriptive commit message in the present tense, such as "Create stylesheet."
+1. On your desktop, create a folder called `techtonica-projects`.  Move your `recipe-page.html` file into `techtonica-projects`.  If you no longer have it, make it again with the Challenge directions from [Phase 1](./phase-1-html-prompt.md).
+2. Use  TextEdit or WordPad to make a new file called `styles.css`. Make sure it is in the `techtonica-projects` directory right next to `recipe-page.html`.
 
 #### Part 2 - Link to your CSS file from within your HTML `<head>` tags
 
-1. Find the `styles.css` file in VSCode that you just created. Add this small bit of CSS code to the file, then save the file.
+1. In `styles.css`, add this small bit of CSS code to the file, then save the file.
```

Comment:
Is there a reason we took out the vscode?

Reply:
Yes, you wanted this to happen before they learn CLI, correct?  CLI is a prereq of VSCode.
https://docs.google.com/spreadsheets/d/1xPgOLI4nEOih3Y5QzpVMLNgDX9g5xjzeE7wfjirWA0M/edit#gid=0

Reply:
I removed Git for the same reason.